src/commands/sourcemaps/__tests__/upload.test.ts

```diff
@@ -46,7 +46,7 @@ describe('upload', () => {
   })
 
   describe('isMinifiedPathPrefixValid: URL without protocol', () => {
-    test('should return false', () => {
+    test('should return true', () => {
```

Comment:
I think this test was "accidentally" passing, because of the leading slash. It should now pass because we check the `host` instead of the `protocol`.